conf/messages

```diff
@@ -202,13 +202,13 @@ gmp.revaluation_rate = Revaluation rate
 gmp.bulk.results = Results
 gmp.bulk.results.csv = CSV
 gmp.bulk.results.reference = Reference {0}
-gmp.bulk.failure.antivirus = The selected file contains a virus. Create the file again and <a href="/guaranteed-minimum-pension/new-bulk-calculation" id="bulk-back-link">upload it</a>.
+gmp.bulk.failure.antivirus = The selected file contains a virus. Create the file again and <a href="/guaranteed-minimum-pension/new-bulk-calculation" id="bulk-back-link">Select a file in CSV format and upload it</a>.
```

Comment:
Change this after contains a virus  to "<a>Create the file again in CSV format and upload it</a>."

Reply:
Or this one maybe we don't even change it cause it wasn't in DAC report